Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "headless_chrome"
-version = "0.1.3"
+version = "0.2.0"
```

Comment:
I think if this gets bumped to `0.2`, we should have docs everywhere. I suggest we bump to `0.1.4` or something